### PR TITLE
sustainability: reword to make it more balanced, towards #66

### PR DESCRIPTION
**Pull request by rhiaro**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -265,13 +265,20 @@ <h3 id="sustainable" data-export="" data-dfn-type="dfn">
...
-New web technologies should not make this situation worse.
-We will consider power consumption and the resulting emissions
-when we introduce new technologies to the web.
-This includes maximizing the lifespan of physical devices
-through maintaining compatibility,
-as well as minimizing data storage and processing requirements.
+Web technologies may have overall positive environmental impacts
+(e.g. remote working reducing the need for long-distance travel)
+as well as negative impacts
+(e.g. large server farms consuming enormous amounts of power, or
```
Comment (torgo) [inline index.html]:
```suggestion
(e.g. large power-consuming server farms, or
```

File: index.html [hunk 2]
```diff
@@ -265,13 +265,20 @@ <h3 id="sustainable" data-export="" data-dfn-type="dfn">
...
+a demand for new hardware devices generating a lot of electronic waste).
+These impacts change over time and vary geographically, due to factors like
+availability of renewable energy generation, or recycling facilities.
```
Comment (torgo) [inline index.html]:
```suggestion
These impacts change over time and vary geographically as technologies develop.
```

File: index.html [hunk 3]
```diff
@@ -265,13 +265,20 @@ <h3 id="sustainable" data-export="" data-dfn-type="dfn">
...
-We will consider power consumption and the resulting emissions
-when we introduce new technologies to the web.
-This includes maximizing the lifespan of physical devices
-through maintaining compatibility,
-as well as minimizing data storage and processing requirements.
+Web technologies may have overall positive environmental impacts
+(e.g. remote working reducing the need for long-distance travel)
+as well as negative impacts
+(e.g. large server farms consuming enormous amounts of power, or
+a demand for new hardware devices generating a lot of electronic waste).
```
Comment (torgo) [inline index.html]:
```suggestion
a demand for new hardware devices generating electronic waste).
```

File: index.html [hunk 4]
```diff
@@ -265,13 +265,19 @@ <h3 id="sustainable" data-export="" data-dfn-type="dfn">
...
-when we introduce new technologies to the web.
-This includes maximizing the lifespan of physical devices
-through maintaining compatibility,
-as well as minimizing data storage and processing requirements.
+Web technologies may have overall positive environmental impacts
+(e.g. remote working reducing the need for long-distance travel)
+as well as negative impacts
+(e.g. large power-consuming server farms, or
+a demand for new hardware devices generating electronic waste).
+These impacts change over time and vary geographically as technologies develop.
```
Comment (rhiaro) [inline index.html]:
```suggestion
Web technologies may have overall positive environmental impacts
as well as negative impacts,
and these can change over time and vary geographically as 
both web and environmental technologies develop.
```

File: index.html [hunk 5]
```diff
@@ -265,13 +265,17 @@ <h3 id="sustainable" data-export="" data-dfn-type="dfn">
...
+as well as negative impacts,
+and these can change over time and vary geographically as 
+both web and environmental technologies develop.
+We will endeavor not to do further harm to the environment
+when we introduce new technologies to the web, and keep in mind
+that people most affected by the environmental consequences of new technologies
+may not be those who benefit from the features introduced.
+This includes, but is not limited to,
+minimizing data storage and processing requirements, as well as
+maximizing the lifespan of physical devices through backwards compatibility.
```
Comment (torgo) [inline index.html]:
```suggestion
lowering carbon emissions by minimizing data storage and processing requirements, as well as reducing electronic waste by
maximizing the lifespan of physical devices through backwards compatibility.
```

Comment (rhiaro) [inline index.html]:
```suggestion
This includes, but is not limited to,
minimizing data storage and processing requirements
in order to reduce carbon emissions, as well as
maximizing the lifespan of physical devices through backwards compatibility
to mitigate generation of electronic waste.
```

**Review by torgo — APPROVED** [review]
lgtm

**Comment by rxgrant** [comment]
This merge does not include the key wording that would make it more balanced.  When you closed  #66, you skipped over a [specific suggestion that was offered](/w3ctag/ethical-web-principles/issues/66#issuecomment-989460123) to make it more balanced.  Since that thread is still locked, you're not going to notice any dissent there...

In particular, it's important to not consider sustainability in isolation, when its intent is to allow advancing EWP even though sometimes using more energy.  You cannot achive "not to do further harm to the environment" while eating meat, using grid electricity, riding in a motor vehicle, or living in modern society.  You know that it is impossible except for a few monks.  What you can do is make the world a better place, but you have to be flexible.  Energy renunciation is an absolutely terrible aspiration to consider as guidance for the whole Internet.

What we want to do is... advance EWP while minimizing environmental impact, considering the resulting net emissions and net efficiencies, as well as the other goals achieved, when we introduce new technologies to the web.

**Comment by rhiaro** [comment]
Hi @rxgrant, sorry I hadn't realised #66 was still locked - that has been rectified. Earlier in that thread I explained already why I shied away from "net" terminology. I didn't skip over that suggestion, I [responded directly to it](https://github.com/w3ctag/ethical-web-principles/issues/66#issuecomment-988702212).

> it's important to not consider sustainability in isolation

That is exactly right. The sustainability principle is intended to be read in the context of all of the other principles in the document, which are about ensuring web technologies are used to improve society, not in isolation. We could write into every single principle a note about balancing them will all of the other principles in the document, but to avoid repeating the point, words to that effect are in the introduction.

**Comment by rxgrant** [comment]
@rhiaro Thanks for reminding me about your statement earlier, though in my timeline one cannot respond to a later comment.  ;)  I think you express well in your linked comment from 66 that we can't settle an age-old philosophical impass if EWP is weaponized, but I'm not satisfied with the introduction's disclaimer, partly because EWP has already been weaponized.  

I think it's important that no principle reads like it disallows progress.  I think the principle does currently read as though using energy may harm the Web, and that energy use should be reduced in all cases.  I think that because I've been told that in arguments about this principle.  I consider the redundancy minimal in the example as I proposed it, but am open to further improvement.

**Comment by rhiaro** [comment]
> though in my timeline one cannot respond to a later comment

my mistake, the parts I responded to came from an earlier comment suggestion that you had later edited. I mixed that up, as the later edits hadn't addressed the parts of the suggestion I wasn't comfortable with.

> because EWP has already been weaponized

Weaponization that uses one principle in isolation is a disingenuous use of the EWP in the first place. I definitely want to improve the wording in the intro to mitigate against this kind of thing if possible, though if people are going to do it I suspect they are going to do so regardless of what the intro or any of the principles actually say. I've opened #76 about this.

**Comment by rhiaro** [comment]
> I think the principle does currently read as though using energy may harm the Web, and that energy use should be reduced in all cases.

I'm reading it over and over, and to be honest I don't see it. The principle as now worded mentions positive and negative environmental impact of web technologies, notes that such impacts can change (implicitly eg. due to improvement of renewable energy provision), and only says "minimizing data storage .. reqs" - implicit there being "as far as possible whilst still achieving your other world-improving aims".

Note we're also trying to keep the principles short and balanced in length, which is why a lot of text from my first draft in this PR was removed in review. As I've said before, there are so many ifs and buts related to this but we don't want to turn it into a primer on energy production/consumption, which is a complex topic in itself.

>  I think that because I've been told that in arguments about this principle

This holds even since the recent rewrite?

**Pull request merged**